### DISCUSSION TITLE
Git showing previous commit while pushing in new branch.

### DISCUSSION THREAD

**Discussion by MeetDOD** [VaibhavArora314/StyleShare#384] [Q&A] [answered]
1. After done with the added captcha feature in the branch issue-356.
2. When i work on other issues and creating their PR it is also showing commits for captcha also for every PR I create.
3. SS:

![image](https://github.com/VaibhavArora314/StyleShare/assets/120380073/5c034c7f-8a80-438d-9580-07b02340554a)

![image](https://github.com/VaibhavArora314/StyleShare/assets/120380073/352a2e8a-ffe8-493c-8d13-a2b56d2bb0b0)

and so on,

4. Due to this i am many problems also it is difficult for project admin to do the same.
5. Please help me to solve this, any solution is there let me ASAP because of this i am stuck in between and not moving forward.
6. @VaibhavArora314 @Ultimateutkarsh11 
7. Thank You !

**Comment by VaibhavArora314** [comment]
I am assuming this issue is probably arising because when you are creating a new branch, you are maybe creating it from the current branch in which you were working. Can you try checking out first to the main branch and then create the new branch?

**Reply by MeetDOD** [reply, marked as answer]
Now It is solved @VaibhavArora314 thanks for your help, when i sync branch for 2 to 3 times it solved automatically 🫱🏻‍🫲🏻
